Sources/PortalSwift/Core/Keychain/PortalKeychainTypes.swift

```diff
@@ -0,0 +1,33 @@
+public struct PortalKeychainClientMetadata: Codable {
```

Comment:
Types from the `PortalKeychain.swift` file have been moved into this file for organization purposes.